server/services/pdfService.js

```diff
@@ -45,7 +45,7 @@ module.exports = function createPdfService(logger, licenceService, conditionsSer
     const currentTemplateId = getIn(rawLicence, ['licence', 'document', 'template', 'decision'])
 
     if (offenceCommittedBeforeCutoffDecision === offenceCommittedBeforeFeb2015 && templateId === currentTemplateId) {
-      return rawLicence
+      return
```

Comment:
return null?

Reply:
if we returned a value from here then we would have to explicitly return a value at the bottom.

Reply:
ok